src/doc/rustc/src/SUMMARY.md

```diff
@@ -82,11 +82,8 @@
     - [x86_64-fortanix-unknown-sgx](platform-support/x86_64-fortanix-unknown-sgx.md)
     - [x86_64-unknown-linux-none.md](platform-support/x86_64-unknown-linux-none.md)
     - [x86_64-unknown-none](platform-support/x86_64-unknown-none.md)
-<<<<<<< HEAD
     - [i386-lynx-lynxos178](platform-support/i386-lynx-lynxos178.md)
```

Comment:
this is a target not available upstream, and likely placed here to set it apart from the rest... would keep it at the end for that reason